include/moveitem.h

```diff
@@ -189,57 +189,40 @@ inline int CountItem(const char* pcItemName, int usMin, int usInvSlots)
 /*********************************************************************************/
 // PackFind will fill a CItemLocation passed to it, or NULL if item not found
 //
-
-// finds an open slot in a bag that has items (not empty bags)
+// finds an open slot in a bag or inventory
 CItemLocation* PackFind(CItemLocation* pFreeSlot, ItemClient* pUnequipSlot)
 {
 	unsigned char  ucCurSize = 10;
-	unsigned short usSlot    = 0;
-
-	unsigned short usDecrement = 0;
+	ItemDefinition* pUnequipInfo = pUnequipSlot->GetItemDefinition();
```

Comment:
If this is null, you can avoid the loop instead of checking if it's null in every iteration